app/views/articles/show.html.erb

```diff
@@ -120,25 +140,20 @@
             </div>
           </a>
         <% end %>
-        <h1 class="<%= @article.title_length_classification %>" itemprop="name headline">
-          <%= @article.title %>
-        </h1>
+        <h1 class="<%= @article.title_length_classification %>"></h1>
+
         <h3>
-          <span itemprop="author" itemscope itemtype="http://schema.org/Person">
-            <meta itemprop="url" content="<%= user_url(@user) %>">
-            <a href="/<%= @user.username %>" class="author">
-              <img class="profile-pic" src="<%= ProfileImage.new(@user).get(width: 50) %>" alt="<%= @user.username %> profile image" />
-              <span itemprop="name"><%= @user.name %></span>
-            </a>
-          </span>
+          <a href="/<%= @user.username %>" class="author">
+            <img class="profile-pic" src="<%= ProfileImage.new(@user).get(width: 50) %>" alt="<%= @user.username %> profile image" />
+          </a>
           <% if @user.twitter_username.present? %>
             <a href="http://twitter.com/<%= @user.twitter_username %>"><%= image_tag_or_inline_svg_tag "twitter", width: 18, height: 18 %></a>
           <% end %>
           <% if @user.github_username.present? %>
             <a href="http://github.com/<%= @user.github_username %>"><%= image_tag_or_inline_svg_tag "github", width: 18, height: 18 %></a>
           <% end %>
           <% if @article.published_timestamp.present? %>
-            <time itemprop="datePublished" datetime="<%= @article.published_timestamp %>"><%= @article.readable_publish_date %></time>
```

Comment:
datetime here is important because it's processed by the client side :)

https://github.com/thepracticaldev/dev.to/blob/c1cf1df7a93afa78e47a45701f3dd818dd6d8d7a/app/assets/javascripts/initializers/initializeArticleDate.js#L6-L12